### PR TITLE
DOC: Adding node, OS requirements and naming conventions.

### PR DESCRIPTION
**Pull request by Stradex**
Adding the Node required versions of the project, the OS that were tested for it and the naming conventions.

### CODE REVIEW

File: website/src/pages/contributing/index.tsx [hunk 1]
```diff
@@ -43,6 +43,24 @@ const Index = () => <section className="home">
       static analysis or unit and integration tests.
     </p>
   </div>
+  <div className="card">
+    <h3>Naming conventions</h3>
+    <p>
```
Comment (Idrinth) [inline website/src/pages/contributing/index.tsx:48]: ul, li for lists please

File: website/src/pages/contributing/index.tsx [hunk 2]
```diff
@@ -43,6 +43,24 @@ const Index = () => <section className="home">
...
+      * CamelCase for all variables and classes.
+      * snake-case for constants.
+      * kebab-case for file names.
+    </p>
+  </div>
+  <div className="card">
+    <h3>Requirements</h3>
+    <div>
+      <p>
```
Comment (Idrinth) [inline website/src/pages/contributing/index.tsx:57]: ul, li for lists please

**Review by Idrinth — CHANGES_REQUESTED** [review]
Thanks for helping, just a tiny fix required :)

**Review by Idrinth — APPROVED** [review]

**Comment by Idrinth** [comment]
grafik
there is a space missing here

**Review by Idrinth — CHANGES_REQUESTED** [review]
Small changes see above

**Comment by Stradex** [comment]
> Small changes see above

Should I keep the kebab-case line or that goes away?

**Comment by Idrinth** [comment]
yes, keep the kebab case, and use either pascal case or upper camel case, not both for ease of reading :)

**Review by Idrinth — APPROVED** [review]

**Pull request merged**